packages/schema/lib/schemas/TriggerSchema.js

```diff
@@ -34,6 +35,7 @@ module.exports = makeSchema(
         anyOf: [
           { $ref: BasicPollingOperationSchema.id },
           { $ref: BasicHookOperationSchema.id },
+          { $ref: BasicHookToPollOperationSchema.id },
```

Comment:
I don't think we can hide just this 3rd item from the docs, we'd have to hide the full property (which isn't what we want)